participants/fullstacksherpa/index.js

```diff
@@ -0,0 +1,25 @@
+function getData() {
+  const selectedCondition = document.getElementById("parameterSelect").value;
+  const apiUrl = `https://www.dnd5eapi.co/api/conditions/${selectedCondition}`;
+
+  // Make a GET request using fetch
+  fetch(apiUrl)
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error("Network response was not ok");
+      }
+      return response.json();
+    })
+    .then((data) => {
+      const resultElement = document.getElementById("result");
+      const descPoints = data.desc.map((point) => `<li>${point}</li>`).join("");
+
+      resultElement.innerHTML = `
+        <h3 class="text-center text-3xl">Description for the <span class="font-bold leading-4 tracking-widest text-purple-600">${selectedCondition} </span>condition</h3>
```

Comment:
Do you have any alternative solution here? Instead of innerHTML method ?

Reply:
Is it okay to use document.createElement and appendChild instead of using innerHTML??